src/Jackett.Common/Indexers/GimmePeers.cs

```diff
@@ -161,26 +163,26 @@ private async Task ProcessPage(List<ReleaseInfo> releases, string searchUrl)
                     }
 
                     // Check if the release has been assigned a category
-                    if (row.Cq().Find("td:eq(0) a").Length > 0)
+                    if (row.QuerySelector("td:nth-of-type(1) a") != null)
```

Comment:
assign `row.QuerySelector("td:nth-of-type(1) a")`to a var to reuse in the next line

Reply:
Not sure if ReSharper or built in to VS, but CTRL+R,V (refactor into variable) detected both and made this as simple as naming the variable in one place.

Reply:
I'm using Rider, but thank you anyway!

Reply:
If it's a feature of ReSharper, then it should be integrated in Rider, since that's the same company.